docs/user/tutorial/40-people.rst

```diff
@@ -80,11 +84,40 @@ automatically.
 
 We'll create a *People* page, and add an apphook for the Aldryn People application to it.
 
-.. todo:: screenshots of following steps (if they are new steps)
-
 .. todo:: write up steps properly
 
-* create new page called People
-* add Apphook
-* publish page
-* view page with automatic list of people in it
+1. Create new page called People
+
+.. image:: /user/tutorial/images/create_people_page.png
```

Comment:
Same note as above: *if* it is possible to make these images wider and less tall, we should.